lib/twitter_cldr/js/renderers/numbers/rbnf/rbnf_test_helper_renderer.rb

```diff
@@ -0,0 +1,41 @@
+# encoding: UTF-8
+
+# Copyright 2012 Twitter, Inc
+# http://www.apache.org/licenses/LICENSE-2.0
+
+module TwitterCldr
+  module Js
+    module Renderers
+      module TestHelpers
+
+        class RBNFTestHelperRenderer < TwitterCldr::Js::Renderers::Base
+          set_template "mustache/numbers/rbnf/rbnf_test_helper.coffee"
+
+          def global_resource
+            result = {}
```

Comment:
oh, ok. I'll refactor this once we drop Ruby 1.8. For now I've made it use inject. It's much better than iterating over the loop as I was doing here.